napari_arboretum/io/svg.py

```diff
@@ -14,9 +14,9 @@
 
 
 def svg_line_from_edge(edge: Edge, view_box: tuple) -> str:
-    x1 = ((edge.y[0] - view_box[0]) / view_box[2]) * 100
+    x1 = ((edge.y[1] - view_box[0]) / view_box[2]) * 100
     y1 = ((edge.x[1] - view_box[1]) / view_box[3]) * 100
-    x2 = ((edge.y[1] - view_box[0]) / view_box[2]) * 100
+    x2 = ((edge.y[0] - view_box[0]) / view_box[2]) * 100
```

Comment:
might be worth adding a comment here to explain that `y[1]` and `y[0]` are flipped to be the consistent with the orientation in the vispy plot

Reply:
Done!